ckanext/switzerland/plugins.py

```diff
@@ -545,6 +545,10 @@ def _modify_package_schema(self, schema):
                 },
             }
         )
+        # Ensure license is accepted in resource fields
```

Comment:
I think you added this in the wrong place - this is the `OgdchShowcasePlugin`.

To update the dataset schema, we need to edit `dcat-ap-switzerland_scheming.json`, unless there's a reason that doesn't work?